pom.xml

```diff
@@ -547,6 +547,8 @@
             </excludeModules>
             <excludes>
               <exclude>${shade.prefix}</exclude>
+              <exclude>org.apache.parquet.hadoop.CodecFactory</exclude> <!-- change field type from Configuration to ParquetConfiguration -->
```

Comment:
Just curious: can we remove this in a future PR? Only this PR introduces the breaking change here.

Reply:
I'm not that well versed in how japicmp works exactly, but I reckon this can be removed after the next minor release.